packages/ketcher-core/src/domain/serializers/mol/common.js

```diff
@@ -43,10 +43,13 @@ function parseCTab(/* string */ ctabLines) /* Struct */ {
   )
   const version = countsSplit[11].trim()
   ctabLines = ctabLines.slice(1)
-  if (version === 'V2000') return v2000.parseCTabV2000(ctabLines, countsSplit)
-  else if (version === 'V3000') {
+  if (version === 'V2000') {
+    return v2000.parseCTabV2000(ctabLines, countsSplit)
+  } else if (version === 'V3000') {
     return v3000.parseCTabV3000(ctabLines, !loadRGroupFragments)
-  } else throw new Error('Molfile version unknown: ' + version) // eslint-disable-line no-else-return
+  } else {
+    throw new Error('Molfile version unknown: ' + version) // eslint-disable-line no-else-return
```

Comment:
I think we can get rid of else on lines 48 and 50
This way we won't need an eslint exception

Reply:
Good proposal, thanks